packages/react-components/react-datepicker-compat/src/components/DatePicker/useDatePicker.tsx

```diff
@@ -25,18 +26,18 @@ function isDateOutOfBounds(date: Date, minDate?: Date, maxDate?: Date): boolean
 }
 
 function useFocusLogic() {
-  const textFieldRef = React.useRef<{ focus: () => void }>(null);
+  const inputRef = React.useRef<{ focus: () => void }>(null);
   const preventFocusOpeningPicker = React.useRef(false);
 
   const focus = () => {
-    textFieldRef.current?.focus?.();
+    inputRef.current?.focus?.();
   };
 
   const preventNextFocusOpeningPicker = () => {
     preventFocusOpeningPicker.current = true;
   };
 
-  return [textFieldRef, focus, preventFocusOpeningPicker, preventNextFocusOpeningPicker] as const;
+  return [inputRef, focus, preventFocusOpeningPicker, preventNextFocusOpeningPicker] as const;
```

Comment:
_nit:_ Keep alphabetical ordering

```suggestion
  return [focus, inputRef, preventFocusOpeningPicker, preventNextFocusOpeningPicker] as const;
```